models/keep3r_network/ethereum/keep3r_network_ethereum_view_job_credits_log.sql

```diff
@@ -0,0 +1,123 @@
+{{ config (
+    materialized = 'view',
+    alias = 'view_job_credits_log',
```

Comment:
Can you remove `view` from the alias? We don't want to specify the materialization in the object name in case it changes at some point. This is a naming convention we decided not to port over from v1.